assets/icons/return-delivery-mailbox.svg

```diff
@@ -0,0 +1,7 @@
+<svg id="b459e6f2-21e1-4be6-bd0b-ce8fe05b4b8b" data-name="Export here" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 200 200">
+  <title>Master (Dont Touch!)</title>
```

Comment:
While we have been very poor with using correct titles, try and get them in when we can